.github/workflows/macos_pr.yml

```diff
@@ -102,17 +102,17 @@ jobs:
       matrix:
         flavor: [ "Sandbox", "Non-Sandbox" ]
         include:
-          - scheme: DuckDuckGo Privacy Browser
+          - scheme: macOS Browser
             flavor: Non-Sandbox
-          - scheme: DuckDuckGo Privacy Browser App Store
+          - scheme: macOS Browser App Store
             flavor: Sandbox
           - active-arch: YES
             flavor: Non-Sandbox
           - active-arch: NO
             flavor: Sandbox
-          - integration-tests-target: Integration Tests
+          - integration-tests-target: macOS Integration Tests
```

Comment:
I see that you've renamed integration tests targets (in addition to schemes), so this works. But flagging this in case that rename was unintentional (which may be the case, considering no other target was renamed):

<img width="260" alt="Screenshot 2025-01-31 at 14 04 46" src="https://github.com/user-attachments/assets/9062d45f-dd82-4911-927c-6d85a32f3681" />

Reply:
This is indeed intentional, since it was the only way to get Xcode to rename these as we access them by their target name `integration-tests-target` - there is no `macOS Integration Tests App Store` scheme at all for example.

That said, I'm open to rolling this back since it's a little outside the scope of what this PR is trying to achieve.

Reply:
I see – so we don't need this rename after all, right? We're using the browser scheme and this target is there only to exclude some tests from being executed. I'd roll this back because it now looks a bit weird on the targets list, wdyt?